### PR TITLE
A lot of engi guidebooks translate

### PR DESCRIPTION
**Pull request by lzk228**
Часть #1642

### CODE REVIEW

File: Resources/ServerInfo/Guidebook/Engineering/AccessConfigurator.xml [hunk 1]
```diff
@@ -1,32 +1,32 @@
 <Document>
-# Access Configurator
-The access configurator is a tool used to specify what type of personnel may use certain devices.
+# Конфигуратор доступов
+Конфигуратор доступа это инструмент, позволяющий установить, кто из персонала может использовать те или иные устройства.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/AccessConfigurator.xml:3]:
```suggestion
Конфигуратор доступа - это инструмент, позволяющий установить, кто из персонала может использовать те или иные устройства.
```

File: Resources/ServerInfo/Guidebook/Engineering/AccessConfigurator.xml [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+На каждой станции имеется до двух конфигураторов доступов. Первый находится у старшего инженера, а второй - у главы персонала.
 
-## How to use the access configurator
-To modify a device using the access configurator
-- First, use the access configurator on the chosen device to link them together. This will automatically open the configurator UI.
-- Next, insert an ID card into the access configurator.
-- Set the access requirements of the connected device. What requirements can be added or removed will depend upon the access privileges of the inserted ID card.
-- Any changes made will be applied immediately - simply eject the ID card from the access configurator and close the UI when you are done.
+## Как использовать конфигуратор доступов?
+Чтобы изменить устройство с помощью конфигуратора доступов
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/AccessConfigurator.xml:15]:
```suggestion
Чтобы изменить устройство с помощью конфигуратора доступов:
```

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml [hunk 3]
```diff
@@ -1,76 +1,76 @@
...
-Fortunately, certain countermeasures can installed into airlocks to inconvenience any would be trespassers.
+К счастью, в шлюзах можно установить определённые меры противодействия, чтобы доставить неудобства любым нарушителям.
 
-## Medium security airlocks
-The most basic form of intrusion deterrence is to install internal steel plating that will prevent access to internal wiring of the airlock. 
+## Шлюзы со средней степенью защиты
+Самый простой способ защиты от вторжения - это установка внутренней стальной пластины, которая предотвращает доступ к внутренней проводке шлюза.
 
-To upgrade a basic airlock to a medium security airlock, you will require the following materials
+Чтобы улучшить базовый шлюз до шлюза средней степени защиты, вам понадобятся следующие материалы
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml:10]:
```suggestion
Чтобы улучшить базовый шлюз до шлюза средней степени защиты, вам понадобятся следующие материалы:
```

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml [hunk 4]
```diff
@@ -1,76 +1,76 @@
...
+Чтобы улучшить базовый шлюз,
+- С помощью отвёртки откройте тех. панель шлюза.
+- Добавьте листы стали в шлюз.
+- Приварите стальные листы к шлюзу.
+- Закройте тех. панель с помощью отвёртки.
 
-## High security airlocks
-For airlocks leading to the more sensitive areas of the space station, the use of stronger deterrents are advised. High security airlocks have improved armor plating to protect its internal wiring, along with an electrified security grille. 
+## Шлюзы с высокой степенью защиты
+Для шлюзов, ведущих в более уязвимые зоны космической станции, рекомендуется использовать более надёжные средства защиты. Шлюзы с высокой степенью защиты имеют усиленную пластину для защиты внутренней проводки, а также электрифицированную защитную решетку.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml:30]:
```suggestion
Для шлюзов, ведущих в более уязвимые зоны космической станции, рекомендуется использовать более надёжные средства защиты. Шлюзы с высокой степенью защиты имеют усиленную пластину для защиты внутренней проводки, а также электрифицированную защитную решётку.
```

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml [hunk 5]
```diff
@@ -1,76 +1,76 @@
...
 
-To upgrade a basic airlock to a high security airlock, you will require the following materials
+Чтобы обновить базовый шлюз до шлюза с высокой степенью защиты, вам понадобятся следующие материалы
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml:32]:
```suggestion
Чтобы обновить базовый шлюз до шлюза с высокой степенью защиты, вам понадобятся следующие материалы:
```

File: Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml [hunk 6]
```diff
@@ -1,76 +1,76 @@
...
+- Добавьте металлические стержни в шлюз.
+- Закройте тех. панель с помощью отвертки.
 
-## Maximum security airlocks
-You can optionally upgrade a high security airlock to a maximum security airlock. Maximum security airlocks possess an additional layer of plasteel plating on top of its other protections. 
+## Шлюзы с максимальной степенью защиты
+При желании вы можете повысить степень защиты шлюза до максимальной. Шлюзы максимальной степени защиты имеют дополнительный слой пластального покрытия в дополнение к остальным защитным элементам.
 
-To upgrade a high security airlock to a maximum security airlock, you will require the following materials
+Чтобы переоборудовать шлюз высокой степени защиты в шлюз максимальной степени защиты, вам потребуются следующие материалы
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/AirlockSecurity.xml:58]:
```suggestion
Чтобы переоборудовать шлюз высокой степени защиты в шлюз максимальной степени защиты, вам потребуются следующие материалы:
```

File: Resources/ServerInfo/Guidebook/Engineering/Atmospherics.xml [hunk 7]
```diff
@@ -27,12 +27,12 @@
 <Box>
 <GuideEntityEmbed Entity="PortableScrubber"/>
 </Box>
-# Gas mixes and Burn chambers
-In the event you finish all the tasks at hand, you can make some extra power or money by creating new chemical gasses.
+# Смешивание газов и камеры сгорания
+В случае если вы выполнили все поставленные задачи, вы можете получить дополнительную энергию или деньги, создавая новые химические газы.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/Atmospherics.xml:31]:
```suggestion
В случае, если вы выполнили все поставленные задачи, вы можете получить дополнительную энергию или деньги, создавая новые химические газы.
```

File: Resources/ServerInfo/Guidebook/Engineering/PortableGenerator.xml [hunk 8]
```diff
@@ -1,42 +1,42 @@
...
 
   <Box>
-    <GuideEntityEmbed Entity="PortableGeneratorJrPacman" Caption="J.R.P.A.C.M.A.N." />
+    <GuideEntityEmbed Entity="PortableGeneratorJrPacman" Caption="М.И.Н.И.П.А.К.М.А.Н." />
     <GuideEntityEmbed Entity="WeldingFuelTank" />
   </Box>
 
-  The J.R.P.A.C.M.A.N. can be found across the station in maintenance areas, and is ideal for crew to set up themselves whenever there are power issues.
-  Setup is incredibly easy: wrench it down above an [color=green]LV[/color] power cable, give it some welding fuel, and start it up.
+  М.И.Н.И.П.А.К.М.А.Н. можно найти по всей станции в технических помещениях, и это идеальное решение для экипажа, чтобы установить его самостоятельно, когда возникают проблемы с электричеством.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/PortableGenerator.xml:19]:
```suggestion
  М.И.Н.И.П.А.К.М.А.Н. можно найти по всей станции в технических помещениях и это идеальное решение для экипажа, чтобы установить его самостоятельно, когда возникают проблемы с электричеством.
```

File: Resources/ServerInfo/Guidebook/Engineering/PortableGenerator.xml [hunk 9]
```diff
@@ -1,42 +1,42 @@
...
+  Установить его невероятно просто: закрепите его над [color=green]низковольтным[/color] кабелем, залейте в него сварочное топливо и запустите.
 
-  Welding fuel should be plentiful to find around the station. In a pinch, you can even transfer some from the big tanks with a soda can. Just remember to empty the soda can first, I don't think it likes soda as fuel.
+  Сварочного топлива на станции должно быть много. В крайнем случае, можно даже перелить немного из больших баков с помощью банки из-под газировки. Только не забудьте сначала опорожнить банку из-под газировки, не думаю, что генератору нравится сода в качестве топлива.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/PortableGenerator.xml:22]:
```suggestion
  Сварочного топлива на станции должно быть много. В крайнем случае, можно даже перелить немного из больших баков с помощью банки из-под газировки. Только не забудьте сначала опорожнить банку из-под газировки - не думаю, что генератору нравится газировка в качестве топлива.
```

File: Resources/ServerInfo/Guidebook/Engineering/TEG.xml [hunk 10]
```diff
@@ -1,39 +1,39 @@
 ﻿<Document>
-  # Thermo-electric Engine (TEG)
+  # Термоэлектрический генератор (ТЭГ)
 
-  The TEG generates power by exchanging heat between hot and cold gases. On station, hot gas is usually created by burning plasma, and an array of heat-exchanging pipes in space radiates away heat to make a cold side.
+  ТЭГ вырабатывает энергию за счет теплообмена между горячими и холодными газами. На станции горячий газ обычно образуется в результате сжигания плазмы, а массив теплообменных труб в космосе излучает тепло, чтобы сделать холодную часть.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/TEG.xml:4]:
```suggestion
  ТЭГ вырабатывает энергию за счет теплообмена между горячими и холодными газами. На станции горячий газ обычно образуется в результате сжигания плазмы, а массив теплообменных труб в космосе излучает тепло, чтобы охладить газ.
```

File: Resources/ServerInfo/Guidebook/Engineering/TEG.xml [hunk 11]
```diff
@@ -1,39 +1,39 @@
...
+  Нет предпочтений в том, какая сторона должна быть горячей или холодной, необходима лишь разница в температуре между ними. Газы в двух "контурах" никогда не смешиваются, между ними происходит только обмен энергией. Горячая сторона будет охлаждаться, холодная - нагреваться.
 
-  ## Burn Chamber
+  ## Камера сгорания
 
-  As I'm sure a wise person once said: the best way to make something hot is to light it on fire. Well, depending on context that may not be very wise, but luckily your engineering department has just what's needed to do it wisely after all.
+  Наверняка кто-то из мудрых людей однажды сказал: лучший способ сделать что-то горячим - это поджечь его. В зависимости от контекста это может быть не очень мудро, но, к счастью, у вашего инженерного отдела есть все необходимое для того, чтобы сделать это мудро.
 
-  TODO: somebody fill this out once we settle on a general mapped layout for burn chambers.
+  TODO: кто-нибудь заполните это, когда мы определимся с общей схемой расположения камер сгорания.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/TEG.xml:28]:
```suggestion
  TODO: кто-нибудь, заполните это, когда мы определимся с общей схемой расположения камер сгорания.
```

File: Resources/ServerInfo/Guidebook/Engineering/TEG.xml [hunk 12]
```diff
@@ -1,39 +1,39 @@
...
-  ## Coolant Array
+  ## Охлаждающая система
 
-  A whole bunch of heat-exchanger piping in space. There's not much to say here: gas goes through and cools down.
+  Целая куча теплообменных труб в космосе. Тут и говорить нечего: газ проходит через них и охлаждается.
 
   ## The Pipes
 
-  TODO: somebody fill this out once we settle on a general mapped layout for burn chambers.
+  TODO: кто-нибудь заполните это, когда мы определимся с общей схемой расположения камер сгорания.
```
Comment (Kit0vras) [inline Resources/ServerInfo/Guidebook/Engineering/TEG.xml:36]:
```suggestion
  TODO: кто-нибудь, заполните это, когда мы определимся с общей схемой расположения камер сгорания.
```

**Review by Kit0vras — CHANGES_REQUESTED** [review]

File: Resources/ServerInfo/Guidebook/Engineering/TEG.xml (hunk 10 above)
Comment (lzk228) [reply Resources/ServerInfo/Guidebook/Engineering/TEG.xml:4]: не

**Pull request merged**